_posts/2023-12-22-central-workflows.markdown

```diff
@@ -0,0 +1,197 @@
+---
+layout: post
+title:  "DRY in GitHub Actions with reusable workflows"
+date:   2023-12-22
+description: "GitHub actions are a central piece on our CI/CD system. We've been using it for less than 2 years (migration from bibucket) - but since minute 1 it was clear for us it was fundamental to focus on scalability and usability<br />
+              Before jumping into features and complex scripting, we talked about the arquitecture - how would it look like for a developer to start working on a new repository, and how would it look for us to deal with a new feature or fix on existing workflows<br />
+              Composite actions and reusable workflows were in our radar - but only intra-repo (using the same workflow to deploy to dev, stage and prod on different scenarios). In this post you will find how our journey started, which problems we solved and how does our future look like"
+categories: CI/CD pipelie
+preview_image: /assets/img/posts/reusable_workflows_preview.png
+author: "Roberto Gutiérrez"
```

Comment:
`"Roberto Gutiérrez Sopedra"`

the name needs to match exactly, otherwise the author won't be found